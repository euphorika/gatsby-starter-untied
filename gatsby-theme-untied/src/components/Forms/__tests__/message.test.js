import React from 'react'
import renderer from 'react-test-renderer'
import Message from '../message'

describe('Message', () => {
  it('renders info message correctly', () => {
    const element = <Message>Error Message</Message>
    const tree = renderer.create(element).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders warn message correctly', () => {
    const element = <Message type="warn">Error Message</Message>
    const tree = renderer.create(element).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders error message correctly', () => {
    const element = <Message type="error">Error Message</Message>
    const tree = renderer.create(element).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
