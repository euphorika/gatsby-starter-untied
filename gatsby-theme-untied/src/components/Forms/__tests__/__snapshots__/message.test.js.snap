// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message renders error message correctly 1`] = `
<div
  className="error css-0"
>
  Error Message
</div>
`;

exports[`Message renders info message correctly 1`] = `
<div
  className="info css-0"
>
  Error Message
</div>
`;

exports[`Message renders warn message correctly 1`] = `
<div
  className="warn css-0"
>
  Error Message
</div>
`;
